From ba167c262713c5f3ae46f9da9c02440dcb4da788 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Thu, 4 Jan 2024 12:19:11 +0100
Subject: [PATCH 0366/1695] dts: stm32: add STM32MP13 tamper pinctrl nodes

Add pinctrl for external tamper events.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git bbf4640845a51b1d305f1552b2c5f3239eb32324 dts: stm32: add STM32MP13 tamper pinctrl nodes]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I09ab40aeb28a80f65aad6ac85fbc83dce9c874ab
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349413
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp13-pinctrl.dtsi | 42 ++++++++++++++++++++++++
 1 file changed, 42 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp13-pinctrl.dtsi b/core/arch/arm/dts/stm32mp13-pinctrl.dtsi
index 6bf841987..453e34b92 100644
--- a/core/arch/arm/dts/stm32mp13-pinctrl.dtsi
+++ b/core/arch/arm/dts/stm32mp13-pinctrl.dtsi
@@ -25,6 +25,48 @@
 		};
 	};
 
+	tamp0_in1_pin_a: tamp0_in1_pin_0 {
+		pins {
+			pinmux = <STM32_PINMUX('F', 10, RSVD)>;
+			st,tamp-id = <1>;
+		};
+	};
+
+	tamp0_in2_pin_a: tamp0_in2_pin_0 {
+		pins {
+			pinmux = <STM32_PINMUX('A', 6, RSVD)>;
+			st,tamp-id = <2>;
+		};
+	};
+
+	tamp0_in3_pin_a: tamp0_in3_pin_0 {
+		pins {
+			pinmux = <STM32_PINMUX('C', 0, RSVD)>;
+			st,tamp-id = <3>;
+		};
+	};
+
+	tamp0_in5_pin_a: tamp0_in5_pin_0 {
+		pins {
+			pinmux = <STM32_PINMUX('C', 3, RSVD)>;
+			st,tamp-id = <5>;
+		};
+	};
+
+	tamp0_in8_pin_a: tamp0_in8_pin_0 {
+		pins {
+			pinmux = <STM32_PINMUX('I', 0, RSVD)>;
+			st,tamp-id = <8>;
+		};
+	};
+
+	tamp0_out1_pin_a: tamp0_out1_pin_0 {
+		pins {
+			pinmux = <STM32_PINMUX('I', 0, RSVD)>;
+			st,tamp-id = <1>;
+		};
+	};
+
 	uart4_pins_a: uart4-0 {
 		pins1 {
 			pinmux = <STM32_PINMUX('D', 6, AF8)>; /* UART4_TX */
-- 
2.47.2

